### PR TITLE
feature(REPORT): Bold reports V5.4 Release Changes

### PR DESCRIPTION
**Pull request by Vishnu7101**
<table>
    <tr>
        <td>Feature description</td>
        <td>
To Update the Bold Reports - ASP.NET Core public repo with the  Bold-Reports 5.4 Release Changes</td>
    </tr>
    <tr>
        <td>Analysis</td>
        <td> Analyzed about Bold reports V5.4 release changes</td>
    </tr>
    <tr>
        <td>Reason for not identifying earlier</td>
        <td>
            It's a new feature</td>
    </tr>
    <tr>
        <td>Is it a breaking issue?</td>
        <td>No, It's a new feature </td>
    </tr>
    <tr>
        <td>Solution description</td>
        <td>
<li>To modify the files according to the latest changes made in Bold report V5.4
<li>Included package josn and gulp file in order to download puppeteer.
</td>
    </tr>
    <tr>
        <td>Areas affected and ensured</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test cases</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test bed sample locations</td>
        <td>VS & VS Code</td>
    </tr>
</table>

### CODE REVIEW

**Review by Subhasree-Vasudevan — APPROVED** [review]
Changes Fine

File: wwwroot/css/bold-reports/bold.reportdesigner.core.material.min.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿@charset "UTF-8";
```
Comment (Subhasree-Vasudevan) [inline wwwroot/css/bold-reports/bold.reportdesigner.core.material.min.css:1]: Remove the ej1 scripts and themes

File: wwwroot/css/bold-reports/material/bold.autocomplete.css [hunk 2]
```diff
@@ -0,0 +1,318 @@
+/*-----------------------Map------------------*/
```
Comment (Subhasree-Vasudevan) [inline wwwroot/css/bold-reports/material/bold.autocomplete.css:1]: Remove the unmin files

File: wwwroot/css/bold-reports/bold.reportdesigner.core.material.min.css (hunk 1 above)
Comment (Vishnu7101) [reply wwwroot/css/bold-reports/bold.reportdesigner.core.material.min.css]: Removed

File: wwwroot/css/bold-reports/material/bold.autocomplete.css (hunk 2 above)
Comment (Vishnu7101) [reply wwwroot/css/bold-reports/material/bold.autocomplete.css]: Removed

**Review by karthickthangasamy — APPROVED** [review]

**Pull request merged**